### DISCUSSION TITLE
Old bootstrap version used on docs page doesn't get security updates anymore

### DISCUSSION THREAD

**Discussion by elramus** [encode/django-rest-framework#9028] [Potential Issue]
Just finished up a security audit at my company and we were advised that it was a risk to expose any page still running Twitter Bootstrap < 4. The browsable docs page is on v3.4.1, which came out about 4 and a half years ago. Latest version of Bootstrap is v5.3.

Just curious if there were any plans to update this to a more recent version.

Thanks!

**Comment by DeanElliott96** [comment]
@elramus

I faced a similar issue after a security audit.

To fix this I used the [drf-redesign](https://github.com/youzarsiph/drf-redesign) package which updates the browsable API and uses Bootstrap 5 instead of Bootstrap 3.

It is one of the example ones from the docs, there are also suggestions on how to do this without a package if you don't want to use the package above: https://www.django-rest-framework.org/topics/browsable-api/#third-party-packages-for-customization
